packages/reports/tests/integration/components/filter-builders/date-dimension-test.ts

```diff
@@ -57,6 +57,6 @@ module('Integration | Component | filter-builders/date-dimension', function(hook
 
     assert
       .dom('.filter-builder')
-      .hasText(`${this.filter.displayName} since (>=) Sep 08, 2020`, 'Rendered correctly when collapsed');
+      .hasText(`${this.filter.displayName} since (>=) 09/08/2020`, 'Rendered correctly when collapsed');
```

Comment:
Do we know why this changed?

Reply:
Nope, but across the board all the formatting was coming out in that format rather than the Month Day, year format

Reply:
Before we go forward with this change, we need to understand what is causing the behavior change. @kevinhinterlong any idea here? Will your calendar PR restore the formatting?

Reply:
Yea the calendar PR will restore this to the original behavior. It was updated to temporarily use `format-date-for-granularity` but it will go back so we could skip this change for now